src/Tools/dotnet-monitor/DotNetHost.Windows.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.Diagnostics.Tools.Monitor
+{
+    partial class DotNetHost
+    {
+        private sealed class WindowsDotNetHostHelper : IDotNetHostHelper
+        {
+            public string ExecutableName => "dotnet.exe";
+
+            public bool TryGetDefaultInstallationDirectory(out string dotnetRoot)
+            {
+                // CONSIDER: Account for emulation (emulated path would be under ".\x64\")
+                dotnetRoot = System.IO.Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ProgramFiles), "dotnet");
```

Comment:
will this work for aks + windows containers?

Reply:
I'll check but I believe this should just work.

Reply:
This should work for Windows containers. See the Windows dockerfile from dotnet/dotnet-docker where it is installed to `%ProgramFiles%\dotnet`: https://github.com/dotnet/dotnet-docker/blob/main/src/runtime/6.0/windowsservercore-ltsc2022/amd64/Dockerfile#L26